packages/checkbox/src/react/index.tsx

```diff
@@ -89,60 +122,43 @@ const Checkbox = React.forwardRef((props, forwardedRef) => {
       <input
         readOnly
         ref={ref}
-        tabIndex="-1"
+        tabIndex={-1}
         type="checkbox"
         value={props.value}
-        {...styles.input(themeName, props)}
-        {...filterReactProps(props, { tagName: 'input' })}
+        {...styles.input()}
+        {...omit<CheckboxProps, ['label', 'onCheck', 'error', 'indeterminate']>(
+          props,
+          ['label', 'onCheck', 'error', 'indeterminate']
+        )}
       />
       <div {...styles.label(themeName, props)}>{props.label}</div>
     </label>
   )
-})
-
-Checkbox.propTypes = {
-  checked: PropTypes.bool,
-  disabled: PropTypes.bool,
-  error: PropTypes.bool,
-  indeterminate: PropTypes.bool,
-  label: PropTypes.node.isRequired,
-  name: PropTypes.string,
-  onCheck: PropTypes.func,
-  value: PropTypes.oneOfType([PropTypes.string, PropTypes.number]).isRequired
-}
-
-Checkbox.defaultProps = {
-  checked: false,
-  disabled: false,
-  error: false,
-  indeterminate: false
-}
```

Comment:
are the default props being assigned somewhere. is there why the snapshots changed. do we even need these anymore?

Reply:
My thought that defaults that are false for boolean types are exactly needed anyway.  Undefined is falsy too.